src/main.html

```diff
@@ -169,7 +169,7 @@
             <div id="portsinput">
                 <div class="dropdown dropdown-dark">
                     <select class="dropdown-select" id="port" i18n_title="firmwareFlasherManualPort">
-                        <option value="manual">Manual</option>
+                        <option value="loading">Loading</option>
```

Comment:
Add the `Loading` string to the `messages.json` file.

Reply:
Fixed !